questions/templates/questions/home.html

```diff
@@ -27,5 +27,6 @@
   <body>
     <div class="container" id="content"></div>
     <script src="{% static 'questions/home/app.js' %}" type="text/babel"></script>
+    <noscript>Want to use this website? Turn on JavaScript!</noscript>
```

Comment:
Might be nice if we include a link on how to turn on Javascript, here's one: http://www.enable-javascript.com/